api/proto/info/info.proto

```diff
@@ -801,6 +815,7 @@ message ZInfoApp {
   ZSwState state = 15;
   repeated ZInfoNetwork network = 16;       // up/down; allocated IP
   repeated string volumeRefs = 17;          // volume UUIDs
+  repeated ZfsSnapshotInfo snapshots = 18;  // List with all snapshots for this app
```

Comment:
I want to suggest sending information about snapshots in this message and triggering the sending of the same message when action with snapshots.

Reply:
Well, seems we have mismatch between location of snapshots inside config (inside device options) and info (inside app instance). I think we should have separate type of info messages, but it is discussible.